chapter_09_docker.asciidoc

```diff
@@ -102,7 +100,7 @@ Security and Configuration::
 One way to approach the problem is to get a server,
 and start manually configuring and installing everything,
 hacking about until it works,
-and maybe think about automating things laterfootnote:[
```

Comment:
i know it looks nicer in the asciidoc source if you put the full stop before the `footnote`, but it looks worse in print, so someone will change it back during copyediting...